src/pages/Home/navbar.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { PureComponent } from "react";
```

Comment:
These files do not belong in the Home. Move it to containers. Keep the name of all the component files in sync. Rename this to Navbar.js